app/assets/javascripts/discourse/tests/acceptance/sticky-avatars-test.js

```diff
@@ -0,0 +1,35 @@
+import { acceptance, query } from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+import { settled, visit } from "@ember/test-helpers";
+import { later } from "@ember/runloop";
+
+acceptance("Sticky Avatars", function (needs) {
+  const container = document.getElementById("ember-testing-container");
+
+  needs.hooks.beforeEach(function () {
+    container.scrollTop = 0;
```

Comment:
For whatever reason it seems the ember container isn’t always at the top when running this test so it breaks (on FF).